src/main/resources/templates/db_scripts/migration/extract_credits_from_expenditures.sql

```diff
@@ -17,7 +17,7 @@ UPDATE ${myuniversity}_${mymodule}.budget AS budget
 SET
     jsonb = jsonb || jsonb_build_object(
         'credits', to_jsonb(COALESCE((budget.jsonb->>'credits')::numeric, 0) + credit_sums.total_credits),
-        'expenditures', to_jsonb(COALESCE((budget.jsonb->>'expenditures')::numeric, 0) - credit_sums.total_credits)
+        'expenditures', to_jsonb(COALESCE((budget.jsonb->>'expenditures')::numeric, 0) + credit_sums.total_credits)
```

Comment:
for this one I think we do not need to change script executing from 8.7.0 to 8.7.4 in schema.json to not execute this script twice for customers that already applied 8.7.0 version. 
So we are ok withot changes in schema.json for this file.